frontend/src/components/Navigation/index.jsx

```diff
@@ -21,40 +21,84 @@ class Navigation extends Component {
     render () {
```

Comment:
Update your branch to master, resolve conflicts and i will Aprove this Pull Request